pkgs/applications/networking/irc/weechat/scripts/weechat-matrix/default.nix

```diff
@@ -91,6 +91,7 @@ in buildPythonPackage {
     addToSearchPath program_PYTHONPATH $out/${python.sitePackages}
     patchPythonScript $out/share/matrix.py
     substituteInPlace $out/${python.sitePackages}/matrix/server.py --replace \"matrix_sso_helper\" \"$out/bin/matrix_sso_helper\"
+    substituteInPlace $out/${python.sitePackages}/matrix/uploads.py --replace \"matrix_upload\" \"$out/bin/matrix_upload\"
```

Comment:
```suggestion
    substituteInPlace $out/${python.sitePackages}/matrix/uploads.py --replace-fail \"matrix_upload\" \"$out/bin/matrix_upload\"
```

just noticed the deprecated --replace flag

Reply:
do I make that change across the whole file, or just this line?

Reply:
whole file in a separate commit would be good.